flocker/ca/_ca.py

```diff
@@ -95,10 +99,10 @@ def create_certificate_authority(keypair, dn, request, serial, expiry, digest):
 
     :param int serial: The certificate serial number.
 
-    :param int expiry: Number of seconds from now until this certificate
-        should expire.
+    :param bytes expiry: ASN1 formatted datetime string representing the
```

Comment:
ASN.1 actually has lots of different date/time formats.  This one is a GeneralizedTime.